### PR TITLE
mongoDB integrated

### PR DESCRIPTION
**Pull request by shinesunny**
@varunashok @abhilashsajeev @madhukarmmallia @AmalQB @sam-qburst

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -61,7 +98,7 @@ router.post('/', upload.single('file'), function (req, res, next) {
         };
         params.Key = directoryPath;
         params.Body = stringifiedJson;
-        client.upload(params, function uploadCallback(err, data) { // upload to S3
+        client.upload(params, function uploadCallback(err, data) {
```
Comment (abhilashsajeev) [inline routes/index.js:101]: Handle error. Hope this is done after functionality completion. Merging for now.

Comment (abhilashsajeev) [inline routes/index.js]: Handle error. Hope this is done after functionality completion. Merging for now.

File: routes/index.js [hunk 2]
```diff
@@ -24,7 +25,31 @@ var stringifiedJson;
...
+  MetaTitle: {
+    type: String
+  },
+  MetaDescription: {
+    type: String
+  }
+}, {
+  timestamp: true
+});
+var mongoDataModel = mongoose.model('mongoDataModel', mongodbData);
```
Comment (varunashok) [inline routes/index.js:52]: Please move this schema to a separate file.

Comment (abhilashsajeev) [inline routes/index.js:52]: @shinesunny Do it in code restructure commit.

**Pull request merged**